probe-rs-tools/src/bin/probe-rs/cmd/read.rs

```diff
@@ -69,6 +92,43 @@ impl Cmd {
                 for val in values {
                     print!("{:016x} ", val);
                 }
+                nbytes = self.words * 8;
+            }
+        }
+
+        if let Some(path) = self.output {
```

Comment:
I'm really not sure about the expected behaviour but usually dumping big amounts of data into the terminal is not ideal. What do you think about either printing XOR writing to file?

Reply:
That sounds fine to me! Will implement

Reply:
Done.